.github/workflows/tests_bidi.yml

```diff
@@ -48,6 +48,6 @@ jobs:
       if: ${{ !cancelled() }}
       uses: actions/upload-artifact@v4
       with:
-        name: csv-report
+        name: csv-report-${{ matrix.channel }}
```

Comment:
In the other PR where you created the CSV report I can see that the report is uploaded as zip. Can this be changed so that the .csv is directly available for import? Or will you make a duplicate at some other place so that we can have a stable URL to point to for `IMPORTDATA` in that Google spreadsheet?

Reply:
This is how GitHub [artifacts work](https://github.com/actions/upload-artifact?tab=readme-ov-file#zip-archives). I don't think we can change that. But if you want the file to be uploaded somewhere else, we can add an action for that.

Reply:
If you have a place in mind where to upload it, feel free to send a PR adding an action. We can also try to upload it into blob store similar to how we do with html reports today.

Reply:
https://github.com/microsoft/playwright/pull/34122 should upload each report to the same url in the blob store.